CryptoPkg/Include/Protocol/OneCrypto.h

```diff
@@ -0,0 +1,5446 @@
+/** @file
+  Implements the BaseCryptLib and TlsLib using the services of the EDK II Crypto
+  Protocol/PPI.
+
+  Copyright (C) Microsoft Corporation. All rights reserved.
+  Copyright (c) 2019 - 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+  UPDATED: 2025-Nov-11
+  VERSION: 1.0
+**/
+
+#ifndef ONE_CRYPTO_PROTOCOL_
+#define ONE_CRYPTO_PROTOCOL_
+
+#include <Uefi.h>

```

Comment:
Sean didn't want Uefi.h here. I'm not convinced on that, a protocol is phase specific anyway